egs/TAC/run.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+
+# Exit on error
+set -e
+set -o pipefail
+
+# Main storage directory where dataset will be stored
+storage_dir=$(readlink -m ./datasets)
+
+librispeech_dir=$storage_dir/LibriSpeech
+noise_dir=$storage_dir/Nonspeech
+CONF_FILE=./local/conf.yml
+# After running the recipe a first time, you can run it from stage 3 directly to train new models.
+
+# Path to the python you'll use for the experiment. Defaults to the current python
+# You can run ./utils/prepare_python_env.sh to create a suitable python environment, paste the output here.
+python_path=python
+
+# Example usage
+# ./run.sh --stage 3 --tag my_tag --task sep_noisy --id 0,1
+
+# General
+stage=0  # Controls from which stage to start
+tag=""  # Controls the directory name associated to the experiment
+# You can ask for several GPUs using id (passed to CUDA_VISIBLE_DEVICES)
+id=0
+eval_use_gpu=1
+
+# Dataset option
+dataset_type=adhoc
+
+. utils/parse_options.sh
+
+dumpdir=data/$suffix  # directory to put generated json file
+
+# check if gpuRIR installed
+if ! ( pip list | grep -F gpuRIR ); then
+  echo 'This recipe requires gpuRIR. Please install gpuRIR.'
+  exit
+fi
+
+if [[ $stage -le  0 ]]; then
+  echo "Stage 0: Downloading required Datasets"
+
+  if ! test -e $librispeech_dir/train-clean-100; then
+    echo "Downloading LibriSpeech/train-clean-100 into $storage_dir"
+    wget -c --tries=0 --read-timeout=20 http://www.openslr.org/resources/12/train-clean-100.tar.gz -P $storage_dir
+	  tar -xzf $storage_dir/train-clean-100.tar.gz -C $storage_dir
+	  rm -rf $storage_dir/train-clean-100.tar.gz
+	fi
+
+  if ! test -e $librispeech_dir/dev-clean; then
+    echo "Downloading LibriSpeech/dev-clean into $storage_dir"
+	  wget -c --tries=0 --read-timeout=20 http://www.openslr.org/resources/12/dev-clean.tar.gz -P $storage_dir
+	  tar -xzf $storage_dir/dev-clean.tar.gz -C $storage_dir
+	  rm -rf $storage_dir/dev-clean.tar.gz
+	fi
+
+  if ! test -e $librispeech_dir/test-clean; then
+    echo "Downloading LibriSpeech/dev-clean into $storage_dir"
+	  wget -c --tries=0 --read-timeout=20 http://www.openslr.org/resources/12/test-clean.tar.gz -P $storage_dir
+	  tar -xzf $storage_dir/test-clean.tar.gz -C $storage_dir
+	  rm -rf $storage_dir/test-clean.tar.gz
+	fi
+
+  if ! test -e $storage_dir/Nonspeech; then
+       echo "Downloading Noises into $storage_dir"
+	  wget -c --tries=0 --read-timeout=20 http://web.cse.ohio-state.edu/pnl/corpus/HuNonspeech/Nonspeech.zip -P $storage_dir
+	  unzip $storage_dir/Nonspeech.zip -d $storage_dir
+	  rm -rf $storage_dir/Nonspeech.zip
+  fi
+
+fi
+
+if [[ $stage -le  1 ]]; then
+  echo "Stage 1: Creating Synthetic Datasets"
+
+  #git clone https://github.com/yluo42/TAC ./local/TAC
+  cd local/TAC/data
+  #$python_path create_dataset.py \
+   #             --output-path=$storage_dir \
+#		--dataset=$dataset_type \
+#		--libri-path=$librispeech_dir \
+#		--noise-path=$noise_dir
+cd ../../../	
+fi
```

Comment:
So, we're missing this, right?

Reply:
no i forgot to re-comment it back. It is working